updatehub-sdk/src/error.rs

```diff
@@ -18,4 +18,8 @@ pub enum Error {
     UnexpectedResponse(#[error(not(source))] surf::StatusCode),
 
     Client(#[error(not(source))] surf::Error),
+
+    IO(#[error(not(source))] std::io::Error),
+
+    Ctrlc(#[error(not(source))] async_ctrlc::Error),
```

Comment:
These wrapped up types are actually errors, so we shouldn't use `not(source)` here.